### PR TITLE
Add missing cross-track labels

### PR DESCRIPTION
**Pull request by ErikSchierboom**
The reputation and v3-migration labels are shared across tracks.

This is a draft and should not be merged until we've fixed the labels.

### CODE REVIEW

File: global-files/.github/labels.yml [hunk 1]
```diff
@@ -137,3 +138,21 @@
...
+# a contributor gets for an authored/merged/reviewed PR.
+# See https://github.com/exercism/docs/blob/main/product/reputation.md
+
+- name: "reputation/contributed_code/major"
+  description: "Increases the number of reputation points awarded by this PR"
+  color: "fcfa9f"
+
+- name: "reputation/contributed_code/minor"
+  description: "Decreases the number of reputation points awarded by this PR"
+  color: "fcfa9f"
```
Comment (ErikSchierboom) [inline global-files/.github/labels.yml:152]: @iHiD Thoughts on the color to use? Should we visually differentiate the major/minor reputation labels?

**Review by SaschaMann — APPROVED** [review]

File: global-files/.github/workflows/sync-labels.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ on:
       - .github/labels.yml
       - .github/workflows/sync-labels.yml
   schedule:
-    - cron: 6 6 6 * *
+    - cron: 0 0 1 * *
```
Comment (SaschaMann) [inline global-files/.github/workflows/sync-labels.yml:10]: I still think it's bad luck to not use a number that doesn't mean "everything goes smoothly" here.

Comment (ErikSchierboom) [reply global-files/.github/workflows/sync-labels.yml:10]: Well, I think it's bad luck to use the number of the beast here 🤷

File: global-files/.github/labels.yml [hunk 3]
```diff
@@ -110,20 +116,20 @@
...
+# The `x:status/<value>` label indicates if there is already someone
 # working on the issue
 - name: "x:status/claimed"
   description: "Someone is working on this issue"
   color: "4231af"
 
-# The x:type labels describe how much Exercism knowledge
-# is required by the contributor
+# The `x:type/<value>` labels describe what type of work the contributor
+# will be engaged in
```
Comment (iHiD) [inline global-files/.github/labels.yml:126]: Any reason this is over 2 lines?

File: global-files/.github/workflows/sync-labels.yml (hunk 2 above)
Comment (SaschaMann) [reply global-files/.github/workflows/sync-labels.yml:10]: That's just extra luck! This script is a beast, so it fits even better.

Comment (ErikSchierboom) [reply global-files/.github/workflows/sync-labels.yml:10]: Lol

File: global-files/.github/labels.yml [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
 
-# This file is automatically generated by merging two files:
-# 1. The Exercism-wide labels: defined in https://github.com/exercism/org-wide-files/blob/main/synced-files/.github/labels.yml)
-# 2. The repository-specific labels: defined in a `.appends/.github/labels.yml` file (optional)
+# This file is automatically generated by concatenating two files:
 #
-# If you want to use additional labels besides the Exercism-wide labels,
-# don't edit this file directly but instead create a `.appends/.github/labels.yml`
-# file within this repo.
+# 1. The Exercism-wide labels: defined in https://github.com/exercism/org-wide-files/blob/main/global-files/.github/labels.yml
```
Comment (ynfle) [inline global-files/.github/labels.yml:7]:
```suggestion
# 1. The Exercism-wide labels (defined [here](https://github.com/exercism/org-wide-files/blob/main/global-files/.github/labels.yml))
```

I'm not sure if `:` is the right punctuation here. Just a nitpick though

Comment (ErikSchierboom) [reply global-files/.github/labels.yml:7]: 🤷

**Pull request merged**